vault.gemspec

```diff
@@ -19,6 +19,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_runtime_dependency "aws-sdk"
```

Comment:
Makes sense. 

Do you have a recommendation for how to generate the aws signatures required for the new auth endpoint short of re-implementing a significant amount of logic?